src/styles/_buttons.scss

```diff
@@ -0,0 +1,41 @@
+.btn-light-theme {
```

Comment:
Properties in .btn-light-theme and .btn-dark-theme are almost same. Can you reuse `extend` feature to remove css properties duplication